setup.py

```diff
@@ -89,7 +89,7 @@
     packages=find_packages("src"),
     install_requires=[
         "numpy",
-        "tokenizers == 0.0.11",
+        "tokenizers == 0.4.2",
```

Comment:
As we don't have so many unit tests on tokenizers python's binding for now, I would tend to stick to a specific version that will be tested on the CI. Otherwise it might introduce some flaky tests when releasing new tokenizers versions

Reply:
Well, given that we are still introducing breaking changes pretty often in `tokenizers`, I would strongly advise against that.